sdk/lib/convert/json.dart

```diff
@@ -76,6 +76,22 @@ const JsonCodec json = JsonCodec();
 ///
 /// Shorthand for `json.encode`. Useful if a local variable shadows the global
 /// [json] constant.
+///
+/// Example:
+/// ```dart
+///  var data = {};
+///  data['text'] = 'foo';
+///  data["value"] = 2;
+///  data['status'] = false;
+///  data['extra'] = null;
```

Comment:
`data` can be const and I think it's clearer to assign the values as a map literal, rather than afterwards. 

Also should also use single-quotes strings, it's flagged by some of the strict lints.

Reply:
true, it's clearer that way.